test/server/cards/01_SOR/units/FirstLegionSnowTrooper.spec.js

```diff
@@ -0,0 +1,73 @@
+describe('First Legion Snow Trooper', function() {
+    integration(function() {
+        describe('First Legion Snow Troopers ability', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: ['first-legion-snowtrooper'],
+                    },
+                    player2: {
+                        groundArena: ['yoda#old-master'],
+                    }
+                });
+            });
+
+            it('should have no effect when attacking a non-damaged-unit', function () {
+                // actions
+                this.player1.clickCard(this.firstLegionSnowtrooper);
+                this.player1.clickCard(this.yodaOldMaster);
+
+                // check board state
```

Comment:
This test is simple enough that the explainer comments are optional (feel free to keep them if you find them helpful, just letting you know they aren't strictly necessary)

Reply:
if its all the same to you I'd like to keep them to have some form of structure.

Reply:
Yep fine by me!